woocommerce/woocommerce
# Dependency Extraction Webpack - add @woocommerce/price-format

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Add @woocommerce/price-format to DEWP



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Locally, visit the Homescreen `/wp-admin/admin.php?page=wc-admin`
2. In the console confirm `window.wc.priceFormat` is `undefined`
3. In `plugins/woocommerce-admin/client/homescreen/index.tsx` add the following code
```js
import { getCurrency } from '@woocommerce/price-format';
console.log( getCurrency() );
```
4. Refresh the homescreen and confirm `window.wc.priceFormat` is now defined and the console log shows up with out error.
5. Open `plugins/woocommerce-admin/build/app/index.asset.php` and confrim `wc-price-format` is NOT included as an asset dependency.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add @woocommerce/price-format package.

#### Comment 

</details>
